src/analysisd/state.c

```diff
@@ -35,6 +37,81 @@ static void w_get_queues_size();
  */
 static void w_get_initial_queues_size();
 
+/**
+ * @brief Increment syscheck decoded events counter for agents
+ */
+static void w_inc_syscheck_agent_decoded_events(char * agent_id);
```

Comment:
Add the `agent_id` parameter to the function description. Repeat this in all the functions.

Reply:
Done!